### PR TITLE
feat: enable crowdin 2-way sync

### PR DESCRIPTION
**Pull request by p6l-richard**
## What does this PR do?

- adds the `upload_translations` to the crowdin GH action
- enables a two-way sync: 
- if we change something in the source code, the action uploads & syncs w/ crowdin
- if we change something in crowdin, the action syncs with source code 

Fixes # (issue)

- if we change strings in the .json files directly, they won't sync with crowdin and we might lose these changes in the future by a crowdin download
example:
- [these changes](https://github.com/calcom/cal.com/pull/9334/files) were merged into main, but are not present crowdin:
CleanShot 2023-06-06 at 17 06 54@2x

Crowdin doesn't receive these translations:

CleanShot 2023-06-06 at 17 08 02

This can be overwritten whenever we download from crowdin.

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?
⚠️ Not sure yet ⚠️ 
- [ ] I'm currently inquiring with crowdin about how we can test this before merging into production

## Mandatory Tasks
- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by PeerRich** [comment]
how would we test this? should we merge and see what happens?

**Comment by shivamklr** [comment]
Have we been doing this manually prior to this PR?

**Comment by PeerRich** [comment]
prior to this PR we ONLY accept changes in crowdin, not on github. if you change language files in github, it'll overwrite it from crowdin with the next sync

**Review by PeerRich — APPROVED** [review]

**Review by shivamklr — COMMENTED** [review]
LGTM

**Comment by p6l-richard** [comment]
> i'm voting for merging and seeing what happens and then reverting. seems low risk, no? @p6l-richard
TL;DR: Yes, that was the right call. 

I did test this on `website`, where it worked as expected. Same case here, I just chekced it for an English string we've updated 16 hours ago and it did trigger a new translation for the German locale:
CleanShot 2023-07-13 at 14 40 41@2x


[Here's the respective change](https://github.com/calcom/cal.com/commit/496b03e92f5302941553a2a365f4114b1b6809f3#diff-57de4b8e5231f1d699891d98b0381d56e9e24537b0ca5420d1fba4f0597370bcR1626) made.


Sorry, that I have been unresponsive the past days, I was out sick. 🛌
